### DISCUSSION TITLE
Cloudwatch: add AWS/Cloudfront metrics for Cloudfront Functions

### DISCUSSION THREAD

**Discussion by conorevans** [grafana/grafana#62989] [Data Sources]
Grafana currently has Cloudfront and Cloudfront Lambda@Edge metrics, but not Cloudfront Function metrics: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/viewing-cloudfront-metrics.html#monitoring-console.cloudfront-functions

**Comment by gelicia** [comment]
This was added here https://github.com/grafana/grafana/pull/62990
